src/coordination/raft_state.cpp

```diff
@@ -259,6 +259,40 @@ auto RaftState::GetCoordinatorToCoordinatorConfigs() const -> std::vector<Coordi
   return state_manager_->GetCoordinatorToCoordinatorConfigs();
 }
 
+auto RaftState::RemoveCoordinatorInstance(int coordinator_id) -> void {
+  spdlog::trace("Removing coordinator instance {}.", coordinator_id);
+
+  auto cmd_result = raft_server_->remove_srv(coordinator_id);
+
+  if (cmd_result->get_result_code() == nuraft::cmd_result_code::OK) {
+    spdlog::info("Request for removing coordinator {} from the cluster accepted", coordinator_id);
+  } else {
+    throw RaftRemoveServerException(
+        "Failed to accept request for removing coordinator {} from the cluster with the error code {}", coordinator_id,
+        int(cmd_result->get_result_code()));
+  }
+
+  // Waiting for server to join
+  constexpr int max_tries{10};
+  auto maybe_stop = utils::ResettableCounter<max_tries>();
+  constexpr int waiting_period{200};
```

Comment:
just define the milliseconds right here, not an int that then gets converted to milliseconds